includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -155,23 +158,26 @@
 		<?php if ( wc_tax_enabled() ) : ?>
 			<?php foreach ( $order->get_tax_totals() as $code => $tax ) : ?>
 				<tr>
-					<td class="label"><?php echo $tax->label; ?>:</td>
+					<td class="label"><?php echo esc_html( $tax->label ); ?>:</td>
 					<td width="1%"></td>
-					<td class="total"><?php
-						if ( ( $refunded = $order->get_total_tax_refunded_by_rate_id( $tax->rate_id ) ) > 0 ) {
+					<td class="total">
+						<?php
+						$refunded = $order->get_total_tax_refunded_by_rate_id( $tax->rate_id );
+						if ( ( $refunded ) > 0 ) {
```

Comment:
Why extra brackets here?

Reply:
Removed the extra bracket.

Reply:
This is still `( $refunded )` - why are those there?

Reply:
Please have a look [here].(https://github.com/woocommerce/woocommerce/pull/17776/commits/c909d1a4d34bd43ccc1c49901d9e811da997b629) At this commit I changed the bracket issue. Don't know why GitHub hasn't updated this.

Reply:
Maybe it's not pushed to this PR/branch?

Reply:
No. It looks like it's pushed to the correct branch. Please have a look, it says `rnaby:admin-metaboxes-views-translation-escaping-fix` branch.

Reply:
Maybe it's cached. Ill check again.

Reply:
Definitely there. I just checked out the branch.

Reply:
Is there anything I can do. Though, it says my local and GitHub branch is updated with that code.

Reply:
Commit it again?

Reply:
Commit has been done. Please check again.